proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,567 @@
+   internationaler Kommunikation.
+4. Der Verein respektiert die Privatsphäre der Nutzer der angebotenen
+   Web-Dienste und der veröffentlichten Software und erlaubt ihnen die anonyme
+   Teilnahme oder mit vollständiger Identifizierung.
+5. Die Vereinigung verpflichtet sich zur Transparenz und sammelt keine Daten
+   der Nutzer ohne deren ausdrückliche Kenntnisnahme und Zustimmung.
+6. In Fällen, in denen die Dienste des Vereins von einem Drittanbieter
+   bereitgestellt werden die eine separate Benutzeranmeldung erfordern, gelten
+   die Datenschutzbestimmungen des Drittanbieters.
+
+## 15. Inkrafttreten
+
+Diese Satzung tritt mit Beschlussfassung der Mitgliederversammlung in Kraft.
+
+
+## English Draft
+
+<!-- markdownlint-disable-next-line MD026 -->
+## Articles of Association - Mixxx e.V.
+
+As a non-profit corporation incorporated under German law, Mixxx e.V. is
+required to declare its intentions and by-laws in a formal "Articles of
+Association" document. The original and legally binding version is the German
+version. The following translation is for information only.
+
+## Clarification of Terminology
+
+1. ‘Mixxx’ is the name of the free and open-source DJ software developed and
+   provided by the association without any license fees. It enables artists
+   to creatively mix pieces of music in live performances on stage, at parties
+   and dance classes as well as in radio and online broadcasts. Mixxx offers
+   functions such as beatmatching, crossfading and a variety of sound effects.
+2. ‘Mixxx e.V.’ is the name of the association that developes ‘Mixxx’ and thus,
+   among other things supports the free DJ culture.
+3. Free and Open Source Software (FOSS) refers to computer programs that
+   respect the freedom of the user and the community. It grants everyone the
+   right to run, study, modify and redistribute the software. This promotes
+   the common good as everyone can use the software and adapt it to their
+   needs. FOSS is particularly beneficial for schools, artists and musicians
+   with low budgets or incomes, as it gives them access to powerful tools for
+   education, creativity and productivity without financial barriers.
+4. Developing free and open source software involves research, compilation of
+   principles and concepts, and their testing through programming and testing
+   of free software.
+
+## 1. Association Name, Registered Office, Financial Year
+
+1. The Association is registered as Mixxx e.V.
+2. The registered offices are located in **TODO FILL IN LOCATION** . If no fixed office is
+   established, it will be managed from the residence of the Board member
+   acting as general manager.
+3. It is listed under register number **TODO FILL IN NUMBER** in the **TODO FILL IN LOCATION**
+   District Court’s Register of Associations. It is listed under the
+   appropriate register in the District Court’s Register of Associations.
+5. The financial year is the calendar year.
```

Comment:
“Name, registered office and financial year” of the association's articles of association, Section 57 (1) BGB states that the association must be entered in the register of associations. Furthermore, an association must have a fixed registered office. If the following sentence only refers to the association's address, this should be defined more precisely: “If no fixed office has been established, the administration shall follow the place of residence of the respective board member who is responsible for the management.